auth/role.go

```diff
@@ -56,7 +56,7 @@ func (auth *Authenticator) NewRole(name string, channels base.Set) (Role, error)
 }
 
 // Key prefix reserved for role documents in the bucket
-const RoleKeyPrefix = "_sync:role:"
+const RoleKeyPrefix = base.RolePrefix
```

Comment:
I know it'll make the PR more verbose, but we should probably replace all usages of `RoleKeyPrefix` with `base.RolePrefix` and remove these 'intermediate' constants.